### ISSUE TITLE
Sort order for Notes

### ISSUE THREAD

**Issue by abhisheksingh0196** [notaapp/issues#420]
Currently, it seems like we can only sort notes in ascending order based on three variables - name, created and modified date. For the daily notes folder, where I add things to future days as well - created and modified dates do not work well because it mixes up the actual notes date order. The only thing that can work is name but it only does ascending. Let me know if I'm missing something

![Screenshot 2024-03-12 at 10 50 20 AM](https://github.com/notaapp/talk/assets/97225179/ab6d83ec-d02d-4da7-85e3-ad45f69cecc4)

**Comment by astoilkov** [comment]
Yeah, you are right. We don't have a descending option currently. We should add this.
